packages/react/src/components/Rating/Rating.base.tsx

```diff
@@ -165,8 +168,9 @@ export const RatingBase: React.FunctionComponent<IRatingProps> = React.forwardRe
       <div
         ref={ref}
         className={css('ms-Rating-star', classNames.root, rootSizeClass)}
-        aria-label={!readOnly ? ariaLabel : ''}
+        aria-label={!readOnly ? (ariaLabel ? ariaLabel : '') : ''}
```

Comment:
It's fine for this to be undefined/null rather than an empty string if there's no meaningful name. This logic could be simplified to only have one ternary: `!readOnly ? ariaLabel : undefined`